<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SuserOperations</class>
 <widget class="QDialog" name="SuserOperations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="operations_tab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>151</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="cred_oprns">
    <attribute name="title">
     <string>Cred.s</string>
    </attribute>
    <widget class="QLineEdit" name="passw_lineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>191</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="uname_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>User name</string>
     </property>
    </widget>
    <widget class="QLabel" name="passw_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="uname_lineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>191</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="usr_add_radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>103</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="usr_del_radioButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>103</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="suser_checkBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>88</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Suser</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="mac_oprns">
    <attribute name="title">
     <string>MAC num.s</string>
    </attribute>
    <widget class="QLabel" name="host_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Host</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hname_lineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>191</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="macnum_lineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>191</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="macnum_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MAC num.</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="mac_add_radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>103</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="mac_del_radioButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>103</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="current_content_textBrowser">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Current decrypted content.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="apply_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>82</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="keep_decr_checkBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Keep decrypted file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_file_pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>82</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="done_pushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>82</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
